@jonringer are you sure that is due to the `httplib2` update? The build [already failed on Hydra](https://hydra.nixos.org/build/114198877) and my `nix-review` output looks pretty identical:
```
builder for '/nix/store/hi36j9xv7q0zwyj66lyiqlx8vs4g5935-python3.8-celery-4.4.0.drv' failed with exit code 1; last 10 log lines:

    Traceback (most recent call last):
      File "/nix/store/846fz80fiz97cjyyqzq6zgmsamaahrvx-python3.8-celery-4.4.0/lib/python3.8/site-packages/celery/app/trace.py", line 362, in trace_task
        raise InvalidTaskError(
    celery.exceptions.InvalidTaskError: Task keyword arguments is not a mapping

      warn(RuntimeWarning(

  -- Docs: https://docs.pytest.org/en/latest/warnings.html
  = 2403 passed, 145 skipped, 2 deselected, 40 warnings, 1 error in 99.12s (0:01:39) =
```

My `nix-review` process is still running:
```
[2/367/383 built (48 failed), 2836 copied (16488.0/16499.3 MiB), 4414.5 MiB DL] building python3.7-pytorch-1.2.0 (buildPhase): [1424/2994] Building NVCC (Device) object caffe2/CMakeFiles/torch.dir/__/aten/src/AT 
```

But once it's finished I can also post the output. Currently the build failures seem to be unrelated (though there are way more than expected :o). I'm not sure about the behaviour at run-time but I did quickly test this update on my system via GooBook, Ansible, and NixOps.

Also: Should I switch the target branch to `staging`?

--- a/pkgs/development/python-modules/httplib2/default.nix
+++ b/pkgs/development/python-modules/httplib2/default.nix
@@ -2,11 +2,11 @@
 
 buildPythonPackage rec {
   pname = "httplib2";
-  version = "0.15.0";
+  version = "0.17.0";
 
   src = fetchPypi {
     inherit pname version;
-    sha256 = "a5f914f18f99cb9541660454a159e3b3c63241fc3ab60005bb88d97cc7a4fb58";
+    sha256 = "0qc2vn98a4icp3h04pdhiykddz5q6wfi905f19zfxl26kyjd15ny";
   };
 
   # Needs setting up

